scripts/gen_openapi.sh

```diff
@@ -54,7 +54,6 @@ vault auth enable "kerberos"
 vault auth enable "kubernetes"
 vault auth enable "ldap"
 vault auth enable "oci"
-vault auth enable "oidc"
```

Comment:
there's both [oidc](https://github.com/hashicorp/vault-client-go/blob/8a05221827ef6b091d8e71e1b37fac73a9d751d1/openapi.json#L5616) and [jwt](https://github.com/hashicorp/vault-client-go/blob/8a05221827ef6b091d8e71e1b37fac73a9d751d1/openapi.json#L4335) endpoints 

this would mean breaking changes in the openapi spec correct?

Reply:
We don't really have an official openapi.json spec at the moment, but yes this change will remove `oidc` endpoints for the generated spec and the resulting code (similar to the #19401 change). My understanding is that the `oidc` endpoints are all duplicates of the `jwt` ones.

Reply:
In the docs we refer to it as [JWT/OIDC auth method](https://developer.hashicorp.com/vault/api-docs/auth/jwt). There are also a couple OIDC-specif endpoints available: [auth_url](https://developer.hashicorp.com/vault/api-docs/auth/jwt#oidc-authorization-url-request), [callback](https://developer.hashicorp.com/vault/api-docs/auth/jwt#oidc-callback)